exercises/es/test_01_02_01.py

```diff
@@ -4,7 +4,7 @@ def test():
 
     assert isinstance(
         nlp, spacy.lang.en.English
-    ), "El objeto nlp debería ser un instance de la clase de inglés."
+    ), "El objeto nlp debería ser un instance de la clase `English`."
```

Comment:
I think we should keep this because in the updated course, we're not explicitly introducing the concept of `English`, `German` etc. anymore and just create the `nlp` objects with `spacy.blank`. So this is potentially confusing.

Reply:
Understood!

Reply:
Resolved in commit 783da5a